fdbcli/fdbcli.actor.cpp

```diff
@@ -1906,9 +1906,7 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 				}
 
 				if (tokencmp(tokens[0], "renametenant")) {
-					bool _result = wait(makeInterruptable(renameTenantCommandActor(db, tokens)));
-					if (!_result)
-						is_error = true;
+					wait(makeInterruptable(ManagementAPI::renameTenant(db, tokens[1], tokens[2])));
```

Comment:
I would still probably call `renameTenantCommandActor` to do the argument validation and print any output, but that function will for now mostly be a pass-through to the management API.